### ISSUE TITLE
[QUESTION] Train Loss does not descend

### ISSUE THREAD

**Issue by demoonism** [lightly-ai/lightly-train#353]
### 🤔 What’s your question?

Hi Lightly Team,

I am experimenting [distillation] method with BS 1024, resize (224,224) on a small dataset (~30K images) for yolo11s . Now I've trained about 2000 epochs, but I notice train loss does not decrease (actually it increases a little bit.) I wonder if this is normal for the first few thousand epochs? or should I stop and fix something first...

Regards,
Dani.

**Comment by stegmuel** [comment]
Hi,

Could you share the exact script you used for training? 
Any chances you could share a plot of your loss curves?
DistillationV2's loss curves do not decrease a lot across training, but the loss should not increase.
We use pretty [high learning rates](https://github.com/lightly-ai/lightly-train/blob/f32cd12cd03c9a12d630ef1dd7c8cec4235bf10b/src/lightly_train/_methods/distillationv2/distillationv2.py#L104), you might want to reduce that a bit.
Similarly, we use [very strong augmentations](https://github.com/lightly-ai/lightly-train/blob/main/src/lightly_train/_methods/distillationv2/distillationv2_transform.py), depending on your images you might benefit from [tuning them down](https://docs.lightly.ai/train/stable/train/method_transform_args.html#method-transform-args) as well.

Hope this helps.

**Comment by demoonism** [comment]
Hi @stegmuel ,

Please find the train loss below
<img width="4463" height="2955" alt="Image" src="https://github.com/user-attachments/assets/0a3b3e52-9a36-42f7-9384-b05eb43c72a5" />

I guess I was in accurate in my description that the loss increases. I meant to say it oscillates quite strong.

I also find that there seem to be some nearly-duplicate images in my dataset (a drone patrol dataset, where drones may taking photos at the same spot multiple times), would this cause significant issues during contrastive learning？

Regards,
D.

**Comment by liopeer** [comment]
Hi @demoonism! Your loss curve looks completely normal and the oscillation of the loss is expected. And about your additional question: Near duplicates may indeed hurt performance with contrastive objectives, however our distillation does not use any of those. Usually it should not hurt much, but if you have the resources I would definitely recommend doing a little bit of data curation before pretraining.
